frontend/src/store/modules/app.js

```diff
@@ -41,6 +41,10 @@ export const useAppStore = defineStore("app", {
       return (schoolId) =>
         state.schoolsList.find((school) => schoolId === school.schoolId);
     },
+    getSchoolNameByMincode: (state) => {

```

Comment:
This getter isn't getting school name, it's returning a school object and you're getting the name from that object. We're also moving away from naming our getters to start with "get" and reserving that language for API calls to align with student admin.

Can you rename this to schoolByMincode() and change the variable name for schoolId to mincode since we're not passing in a guid?